plugins/woocommerce/templates/emails/email-order-details.php

```diff
@@ -71,15 +71,13 @@
 
 <div style="margin-bottom: <?php echo $email_improvements_enabled ? '24px' : '40px'; ?>;">
 	<table class="td font-family <?php echo esc_attr( $order_table_class ); ?>" cellspacing="0" cellpadding="6" style="width: 100%;" border="1">
-		<?php if ( ! $email_improvements_enabled ) { ?>
 		<thead>
 			<tr>
 				<th class="td" scope="col" style="text-align:<?php echo esc_attr( $text_align ); ?>;"><?php esc_html_e( 'Product', 'woocommerce' ); ?></th>
 				<th class="td" scope="col" style="text-align:<?php echo esc_attr( $text_align ); ?>;"><?php esc_html_e( 'Quantity', 'woocommerce' ); ?></th>
```

Comment:
Quantity should also be aligned right 

<img width="626" height="205" alt="Image" src="https://github.com/user-attachments/assets/4d23b932-8478-4349-9932-8eab9102fb8b" />